java/arcs/core/data/expression/Evaluator.kt

```diff
@@ -23,10 +23,10 @@ import arcs.core.util.Time
  * reflected as exceptions.
  */
 class ExpressionEvaluator(
-    val currentScope: Scope = CurrentScope<Any?>(),
+    val rootScope: Scope = CurrentScope<Any?>(),
```

Comment:
It doesn't look like you use rootScope anywhere. Now that you pass context directly in visit() you can probably remove this.

Reply:
Done